community/modules/scripts/ramble-execute/main.tf

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  commands_content = indent(4, yamlencode(var.commands))
+
+  execute_contents = templatefile(
+    "${path.module}/templates/ramble_execute.yml.tpl",
+    {
+      spack_path  = var.spack_path
+      ramble_path = var.ramble_path
+      log_file    = var.log_file
+      commands    = local.commands_content
+    }
+  )
+
+  previous_ramble_runner_content = var.ramble_runner == null ? "" : var.ramble_runner["content"]
```

Comment:
We should add validation that:
1. The previous runner is ansible
2. That the previous runner has content (runners can also have a file instead)